plugins/CoreVisualizations/CoreVisualizations.php

```diff
@@ -39,23 +38,6 @@ public function deleteUser($userLogin)
         ViewDataTableManager::clearUserViewDataTableParameters($userLogin);
     }
 
-    public function addViewDataTable(&$viewDataTable)
-    {
-        // Both are the same HtmlTable, just the Pivot one has some extra logic in case Pivot is used. 
-        // We don't want to use the same HtmlTable twice in the UI. Therefore we always need to remove one.
-        if (Common::getRequestVar('pivotBy', '')) {
```

Comment:
This actually caused the problem, as for an initially loaded report no request vars are set.